src/genie/libs/parser/nxos/show_track.py

```diff
@@ -0,0 +1,304 @@
+"""show_track.py
+NXOS parser for the following show commands:
+* 'show track'
+"""
+
+import re
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any, Optional
+
+# =======================
+# Schema for 'show track'
+# =======================
+class ShowTrackSchema(MetaParser):
+    """ Schema for 'show track' """
+    schema = {
```

Comment:
we have similar parser in iosxe. so if possible, better to have similar/same schema. can you check this?
https://github.com/CiscoTestAutomation/genieparser/blob/master/src/genie/libs/parser/iosxe/show_track.py#L17

Reply:
Hi @tahigash, I did go through the scemea of showtrack from already existing IOSXE os, but that schema will only cover the track parsing for the interface and IP Route track type wherein for the NXOS OS showtrack we are trying to include all the possible track type like IP SLA, IPv6 Route, List. This will make the schema design different from the existing IOSXE

Reply:
ok. as long as schema makes sense, ok.